examples/analog_in/ai_temperature_signal_read.py

```diff
@@ -0,0 +1,22 @@
+﻿"""Example of analog input temperature acquisition.
+
+This example demonstrates how to acquire thermocouple measurement using
+software timing.
+"""
+
+import pprint
+
+import nidaqmx
+from nidaqmx.constants import ThermocoupleType, TemperatureUnits
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_thrmcpl_chan(
+        "Dev1/ai0", units=TemperatureUnits.DEG_C, thermocouple_type=ThermocoupleType.K
+    )
+
+    print("Reading Data: ")
```

Comment:
A few things about these prints seem weird to me.

- They are sentence fragments, but they are also in title case. Sentence case would make more sense to me.
- They describe what the program is about to do, before it does that, but they include a colon indicating the result.
- They print a space between the colon and newline. A space is only appropriate if you are omitting the newline.

What do the ANSI C examples do?

```
	DAQmxErrChk (DAQmxReadAnalogScalarF64(taskHandle,10.0,&value,0));

	printf("Acquired reading: %f\n",value);
```

Can we do something like that instead? We don't need pprint at all when printing only one sample. 
```
print(f"Acquired reading: {data}")
```

To print multiple samples, we could continue to print the data on a separate line, or we could use pformat to print on the same line:
```
# either
print("Acquired data:")
pp.print(data)

# or
print(f"Acquired data: {pp.pformat(data)}")
```

(It's also possible to use `print("...", end="")` to omit the newline, but I think pformat and f-strings are more readable.)